### PR TITLE
Update VoxelShape Generators to Actual MojMaps and added Yarn.

### PR DESCRIPTION
**Pull request by Nyfaria**
It was still using MCP class names, Mojang Class names are now Available for use (have been for a while), so I updated it to use the Mojang Class names, and Added Yarn.

### CODE REVIEW

**Comment by Nyfaria** [comment]
@Spectre0987 I made this PR and just want to let you know.

File: plugins/voxel_shape_generator.js [hunk 1]
```diff
@@ -5,10 +5,10 @@
 
     Plugin.register('voxel_shape_generator', {
         title: 'VoxelShape Generators',
-        author: 'Spectre0987',
-        description: 'Generates Voxel Shapes (Supports Mojang Mappings and MCP)',
+        author: 'Spectre0987, Nyfaria',
+        description: 'Generates Voxel Shapes (Supports MojMaps, MCP and Yarn)',
         icon: 'bar_chart',
-        version: '0.1.0',
+        version: '0.2.0',
```
Comment (JannisX11) [inline plugins/voxel_shape_generator.js:11]: Make sure to update the version number and general metadata in plugins.json as well.

File: plugins/voxel_shape_generator.js [hunk 2]
```diff
@@ -42,9 +42,10 @@ function openMappingsBox(){
                 type: 'select',
                 options: {
                     momap: 'Mojang Mappings',
-                    mcp: 'MCP'
+                    mcp: 'MCP',
+                    yarn: 'Yarn'
                 },
-                default: 'MCP'
+                default: 'Mojang Mappings'
```
Comment (JannisX11) [inline plugins/voxel_shape_generator.js:48]: This should point to the key, not the value. So "momap". You can also omit it entirely, that way the first option will be the default.

**Review by JannisX11 — CHANGES_REQUESTED** [review]

**Comment by Nyfaria** [comment]
I have made requested changes

**Pull request merged**